docs/how_to_make_a_tahoe-lafs_release.org

```diff
@@ -1,80 +1,97 @@
+  - [ ] made sure they are tagged/labeled
+  - [ ] merged all release PRs
+
+* basic quality checks [0/3]
+  - [ ] all travis CI checks pass
+  - [ ] all appveyor checks pass
+  - [ ] all buildbot workers pass their checks
+
+* freeze master branch [0/]
+  - [ ] announced the freeze of the master branch on IRC (i.e. non-release PRs won't be merged until after release)
+
+* sync documentation [0/7]
   - [ ] NEWS.rst: summarize user-visible changes, aim for one page of text
-  - [ ] update doc files
-    - NEWS.rst: Add final release name and date to top-most item in NEWS.
-    - relnotes.txt
-    - CREDITS
-    - docs/known_issues.rst
-  - [ ] change docs/INSTALL.rst to point to just the current
-    tahoe-lafs-X.Y.Z.tar.gz source code file
-  - [ ] announce the tree is locked on IRC
-  - [ ] git pull, should be empty
-  - [ ] git tag -s -u 68666A7A -m "release Tahoe-LAFS-X.Y.Z" tahoe-lafs-X.Y.Z
-    - produces a "signed tag"
-  - [ ] build locally to make sure the release is reporting itself as the
-    intended version, make tarballs too (for comparison only)
-  - [ ] push tag to trigger buildslaves. Making a code change is no longer
-    necessary (TODO: pushing just 1.12.0b2 was insufficient: travis fired,
-    but not buildbot)
-    - git push official master TAGNAME
-    - that will build tarballs
-  - [ ] make sure buildbot is green
-  - [ ] download tarballs+wheels
-  - [ ] announce tree is unlocked
-  - [ ] compare tarballs+wheels against local copies (but sign upstreams)
-  - [ ] sign each with "gpg -ba -u 68666a7a FILE"
-  - [ ] twine upload dist/*
-  - [ ] test "pip install tahoe-lafs" (from PyPI)
-  - [ ] upload *.asc to org ~source/downloads/
-  - [ ] copy the release tarball,sigs to tahoe-lafs.org: ~source/downloads/
-  - [ ] move old release out of ~source/downloads (to downloads/old/?)
-  - [ ] send out relnotes.txt:
-    - add prefix with SHA256 of tarballs, release pubkey, git revhash
-    - GPG-sign the email with release key
-    - send to tahoe-announce@tahoe-lafs.org and tahoe-dev@tahoe-lafs.org
-  - [ ] update Wiki front page: version on download link, News column
-  - [ ] update Wiki "Doc": parade of release notes (with rev of NEWS.rst)
-  - [ ] close the Milestone on the trac Roadmap
-  - [ ] tweet to @tahoelafs
-  - other stuff:
-  - [ ] update https://tahoe-lafs.org/hacktahoelafs/
-  - [ ] make an "announcement of new release" on freshmeat
+  - [ ] added final release name and date to top-most item in NEWS.rst
+  - [ ] updated relnotes.txt
+  - [ ] updated CREDITS
+  - [ ] updated docs/known_issues.rst
+  - [ ] docs/INSTALL.rst only points to current tahoe-lafs-X.Y.Z.tar.gz source code file
+  - [ ] updated https://tahoe-lafs.org/hacktahoelafs/
+
+* sign + build the tag [0/8]
+
+  - [ ] code passes all checks / tests (i.e. all CI is green)
+  - [ ] documentation is ready (see above)
+  - [ ] git tag -s -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A -m "release Tahoe-LAFS-X.Y.Z" tahoe-lafs-X.Y.Z
```

Comment:
Since the reality is that _Brian_ has to sign the tag, maybe worth saying that here?  Unless we have a plan to change that.